### PR TITLE
Update 10-kfongISD.md file

### PR DESCRIPTION
**Pull request by kfongISD**
updated my file with my caption

### CODE REVIEW

File: _slides/10-kfongISD.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+New caption here
```
Comment (szaryanISD) [inline _slides/10-kfongISD.md:6]: Just New?

Comment (szaryanISD) [reply _slides/10-kfongISD.md:6]: how about 🐱  🐙 🍵 ?

Comment (kfongISD) [reply _slides/10-kfongISD.md:6]: ok 😜

Comment (szaryanISD) [reply _slides/10-kfongISD.md:6]: also forgot to suggest to change your title oops!

**Pull request merged**